packages/vite-plugin-cloudflare/README.md

```diff
@@ -57,7 +76,7 @@ export default {
 };
 ```
 
-You can now develop (`vite dev`), build (`vite build`), preview (`vite preview`), and deploy (`wrangler deploy`) your application.
+You can now develop (`npm run dev`), build (`npm run build`), preview (`npm run preview`), and deploy (`npm exec wrangler deploy`) your application.
```

Comment:
Can also use `npx`...

```suggestion
You can now develop (`npm run dev`), build (`npm run build`), preview (`npm run preview`), and deploy (`npx wrangler deploy`) your application.
```

Reply:
You can but that will download and execute a package if it's missing so you can end up using a different version by mistake.

Reply:
I did also consider adding it as another `package.json` script but deploy isn't a great script name as it is a naming collision in pnpm.